google/cloud/internal/streaming_read_rpc_logging_test.cc

```diff
@@ -100,6 +101,22 @@ TEST_F(StreamingReadRpcLoggingTest, Read) {
   EXPECT_THAT(log_lines, Contains(HasSubstr("Invalid argument.")));
 }
 
+TEST_F(StreamingReadRpcLoggingTest, FormatMetadata) {
+  struct Test {
+    StreamingRpcMetadata metadata;
+    std::string expected;
+  } cases[] = {
+      {{}, ""},
+      {{{"a", "b"}}, "{a: b}"},
+      {{{"a", "b"}, {"k", "v"}}, "{a: b}, {k: v}"},
+  };
+  for (auto const& test : cases) {
+    SCOPED_TRACE("Testing for <" + test.expected + ">");
```

Comment:
This seems redundant in this specific case of comparing strings.

Reply:
Yup, removed.